### PR TITLE
attach expected output of the Databricks example in README.Rmd

### PR DESCRIPTION
**Pull request by yl790**
as @falaki suggested -- let me know if anything is missing

### CODE REVIEW

**Comment by javierluraschi** [comment]
The README describes `Databricks Connect` which happens to be different than the standard `Databricks` connection. `method = "databricks"` has worked for many releases in databricks.com. However, the new feature here is "Databricks Connect" which works by connecting to datbaricks.com from a Desktop computer. So if anything, we would need a screenshot of RStudio Desktop connected to a Databricks cluster.

@lairj09 might already have a screenshot of this we can use.

**Comment by falaki** [comment]
Yes, this new section is about running code on your desktop that sends queries to the Databricks cluster (for remote execution.

**Comment by yl790** [comment]
Alright we got the correct screenshot now. You can preview it at https://github.com/yl790/sparklyr/tree/render_databricks_example_output#connecting-through-databricks-connect

File: README.Rmd [hunk 1]
```diff
@@ -400,17 +400,21 @@ To use sparklyr with Databricks Connect first launch a Cluster on Databricks. Th
...
-```{r eval=FALSE}
-sc <- spark_connect(method = "databricks")
+```{r, eval=FALSE}
+spark_home <- system("databricks-connect get-spark-home")
+sc <- spark_connect(method = "databricks",
+                    spark_home = spark_home)
 copy_to(sc, iris, overwrite = TRUE)
 ```
 
+<img src="tools/readme/databricks-connect.png" class="screenshot" width=750 />
```
Comment (falaki) [inline README.Rmd:416]: Is this screenshot included in this PR?

File: README.Rmd [hunk 2]
```diff
@@ -400,17 +400,21 @@ To use sparklyr with Databricks Connect first launch a Cluster on Databricks. Th
...
 export SPARK_HOME=`databricks-connect get-spark-home`
 ```
 
 Now simply create a spark connection as follows
-```{r eval=FALSE}
-sc <- spark_connect(method = "databricks")
+```{r, eval=FALSE}
+spark_home <- system("databricks-connect get-spark-home")
+sc <- spark_connect(method = "databricks",
+                    spark_home = spark_home)
```
Comment (falaki) [inline README.Rmd:412]: Do you need to set `spark_home` if `SPARK_HOME` is already set?

File: README.Rmd (hunk 1 above)
Comment (yl790) [reply README.Rmd:416]: Yes indeed -- It's also shown in  https://github.com/yl790/sparklyr/tree/render_databricks_example_output#connecting-through-databricks-connect (the preview of what will be rendered after this change)

File: README.Rmd (hunk 2 above)
Comment (yl790) [reply README.Rmd:412]: Not really -- but this was what's shown on the  screenshot provided so I figured the code here should match the screenshot

**Comment by falaki** [comment]
Databricks Connect tests succeeded. View logs [here](https://sparklyr-jenkins.s3.amazonaws.com/logs/jenkins-sparklyr-PR-2417-8/36e55469-7fe4-46a1-8494-ba6857e7f8d5.txt).

**Comment by falaki** [comment]
Databricks Connect tests succeeded. View logs [here](https://sparklyr-jenkins.s3.amazonaws.com/logs/jenkins-sparklyr-PR-2417-9/81c1b387-f759-4bd4-836f-fac01e4c141a.txt).

**Comment by falaki** [comment]
Databricks Connect tests succeeded. View logs [here](https://sparklyr-jenkins.s3.amazonaws.com/logs/jenkins-sparklyr-PR-2417-10/7e127d45-0eb4-4fc5-8602-33aec5da89e1.txt).

**Comment by falaki** [comment]
Databricks Connect tests succeeded. View logs [here](https://sparklyr-jenkins.s3.amazonaws.com/logs/jenkins-sparklyr-PR-2417-11/da6c980a-47cc-4864-bebf-de28a48998c3.txt).

**Comment by falaki** [comment]
Databricks Connect tests succeeded. View logs [here](https://sparklyr-jenkins.s3.amazonaws.com/logs/jenkins-sparklyr-PR-2417-12/0dfc527e-afcf-4ad0-ad5e-b0a8700fa9f0.txt).

**Pull request merged**